--- a/package/kernel/linux/modules/fs.mk	2023-05-28 14:07:54.201670308 +0800
+++ b/package/kernel/linux/modules/fs.mk	2023-07-10 19:06:31.038423494 +0800
@@ -92,9 +92,12 @@ define KernelPackage/fs-cifs
 	CONFIG_CIFS_DFS_UPCALL=n \
 	CONFIG_CIFS_UPCALL=n
   FILES:= \
-	$(LINUX_DIR)/fs/smbfs_common/cifs_arc4.ko@ge5.15 \
-	$(LINUX_DIR)/fs/smbfs_common/cifs_md4.ko@ge5.15 \
-	$(LINUX_DIR)/fs/cifs/cifs.ko
+	$(LINUX_DIR)/fs/smb/common/cifs_arc4.ko@ge6.1 \
+	$(LINUX_DIR)/fs/smb/common/cifs_md4.ko@ge6.1 \
+	$(LINUX_DIR)/fs/smb/client/cifs.ko@ge6.1 \
+  	$(LINUX_DIR)/fs/smbfs_common/cifs_arc4.ko@eq5.15 \
+	$(LINUX_DIR)/fs/smbfs_common/cifs_md4.ko@eq5.15 \
+	$(LINUX_DIR)/fs/cifs/cifs.ko@lt6.1
   AUTOLOAD:=$(call AutoLoad,30,cifs)
   $(call AddDepends/nls)
   DEPENDS+= \
